README.md

```diff
@@ -1 +1,17 @@
-# Design_Your_Own_Grid_Based_Framework
\ No newline at end of file
+# Design-your-own-grid-based-framework
+
+Microverse
+
+This is the eighth project of the HTML and CSS Curriculum at [Microverse](https://www.microverse.org/) - @microverseinc
+* This assignment consists of building a grid-based framework (similar to bootstrap) that includes some basic functionality necessary to build a website.
+* The project was completed using simple HTML and SCSS.
+
+#### [Assignment link](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-your-own-grid-based-framework)
+
+#### [Live Demo](https://rawcdn.githack.com/noomdalv/Design_Your_Own_Grid_Based_Framework/bcfb376efd7d9808bae26f757cff0e5f438ff4b7/index.html)
+
+#### Authors
+
+* [@SofiBretz](https://github.com/SofiBretz)
+* [@noomdalv](https://github.com/noomdalv)
+
```

Comment:
#### `REQUIRED`

If I am to use this :new:  CSS Grid framework. How can I use it? :thinking: 

> Remember about descriptive README file.